### PR TITLE
[Spike] Setup `gov-relayer` app

### PR DESCRIPTION
**Pull request by ken-futureverse**
## Summary

Create new `gov-relayer` app and migrate the new proposal pub/sub over

## Changes

- Use `yargs` to handle commands routing
- Add `proposal-pub` and `proposal-sub` commands
- Set to only store proposal data not delivering message data to simplify the database

## Notes

Check `README.md` for instructions to run on local machine, keen to hear if you have any issues when following the steps @aidan-starke 

I also have to trim down the `proposal/[pid].tsx` temporarily to make the build work as I used different Proposal schema

### CODE REVIEW

File: libs/service/rabbitmq/src/getAMQClient.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { AMQPClient } from "@cloudamqp/amqp-client";
+
+let client: AMQPClient;
+
+export const getAMQClient = async (serverUri: string): Promise<AMQPClient> => {
+	if (client) return client;
```
Comment (aidan-starke) [inline libs/service/rabbitmq/src/getAMQClient.ts]: nitpick, should be `getAMQPClient` but I'd prefer `getRabbitClient` and `getMongoClient` for easier typing

File: libs/node/utils/src/handleNewProposalMessage.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			{ proposalId }
+		);
+
+		if (abortSignal.aborted) return;
+		logger.info("Proposal #%d: 🎾 fetch info [1/3]", proposalId);
+		fetchProposalInfo(api, proposalId)
+			.then((proposalInfo) => {
+				if (abortSignal.aborted) return;
+				if (!proposalInfo) return true;
+				logger.info("Proposal #%d: 📮 file to DB [2/3]", proposalId);
```
Comment (aidan-starke) [inline libs/node/utils/src/handleNewProposalMessage.ts:52]: maybe return false?

**Review by aidan-starke — DISMISSED** [review]

File: libs/node/utils/src/handleNewProposalMessage.ts (hunk 2 above)
Comment (ken-futureverse) [reply libs/node/utils/src/handleNewProposalMessage.ts:52]: yeah actually it's meant to return `true` early here so the message gets clear out of the queue and don't bother retrying

File: libs/service/rabbitmq/src/getAMQClient.ts (hunk 1 above)
Comment (ken-futureverse) [reply libs/service/rabbitmq/src/getAMQClient.ts]: fair point

**Review by aidan-starke — APPROVED** [review]

**Pull request merged**